client/state/themes/constants.js

```diff
@@ -1,8 +1,10 @@
+export const DEFAULT_THEME_QUERY_LIMIT = 1000;
+
 export const DEFAULT_THEME_QUERY = {
 	search: '',
 	tier: '',
 	filter: '',
-	number: 20,
+	number: DEFAULT_THEME_QUERY_LIMIT,
```

Comment:
One side effect of setting 1000 as the default query number is that we will [generate 1000 placeholders](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/themes-list/index.jsx#L92) as well, which can look a bit silly 😅.

Reply:
Ah, nice catch!! 😂 Hmm.... I think we can just hardcode the placeholder count to... 20? Currently 20 is already larger than the number of premium themes anyway 🙈

Reply:
Hardcoded to 20 :)